### PR TITLE
point to KEDA, add a link to ko

### PR DESCRIPTION
**Pull request by vaikas**

### CODE REVIEW

File: broker/README.md [hunk 1]
```diff
@@ -35,11 +35,13 @@ kubectl create secret generic rokn-rabbitmq-broker-secret \
     --from-literal=brokerURL="amqp://$R_USERNAME:$R_PASSWORD@rokn-rabbitmq-client.default:5672"
 ```
 
-install KEDA as documented [here](https://keda.sh/docs/latest/deploy/).
+You can also optionally install [KEDA based
+autoscaler](https://github.com/knative-sandbox/eventing-autoscaler-keda).
```
Comment (mattmoor) [inline broker/README.md:39]: Format markdown:
```suggestion
You can also optionally install
[KEDA based autoscaler](https://github.com/knative-sandbox/eventing-autoscaler-keda).
```

File: broker/README.md [hunk 2]
```diff
@@ -35,11 +35,13 @@ kubectl create secret generic rokn-rabbitmq-broker-secret \
...
 
 ## Installation
 
-install the broker-controller from this repository:
+You must have [`ko`](https://github.com/google/ko) installed.
+Then install the broker-controller from this repository:
```
Comment (mattmoor) [inline broker/README.md:44]: Format markdown:
```suggestion
You must have [`ko`](https://github.com/google/ko) installed. Then install the
broker-controller from this repository:
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**